sagemathinc/cocalc
# preflight: ban specific chrome 77

# Description
* extracting exact chrome version as a quadruple of integers
* testing them against what's in the chromium bugreport

# Testing Steps
I tweaked the numbers in the test to ban myself. Numbers are also parsed correctly.

I should check with browserstack edge, FF, etc. ... but I don't see how this could break.

Did the above, looks good. Money shot:

![Screenshot from 2019-10-08 15-00-56](https://user-images.githubusercontent.com/207405/66397771-d0670480-e9dc-11e9-96ea-a0d80232822b.png)


# Relevant Issues

 #4136

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
